### ISSUE TITLE
TextBox should set HelpText accessible property when it contains placeholder text

### ISSUE THREAD

**Issue by dmitrii-drobotov** [dotnet/winforms#7815]
### .NET version

8.0.100-alpha.1.22423.9

### Did it work in .NET Framework?

No

### Did it work in any of the earlier releases of .NET Core or .NET 5+?

No

### Issue description

From [Edit docs](https://learn.microsoft.com/en-us/windows/win32/winauto/uiauto-supporteditcontroltype):

> If an edit control contains placeholder text (for example, a cue banner), the text should be used as the HelpText property unless the text can be edited by the user and then reused as placeholder text. For example, the Windows Internet Explorer address bar contains the text "about:Tabs" when a new tab is opened. This is not HelpText because it is a programmatic address that can be used or edited by the user.

There is a `PlaceholderText` property on a `TextBox` which acts as a placeholder. It cannot be edited by the user and disappears when `TextBox` gets focus, thus needs to be exposed as a `HelpText`.

### Current behavior:

When setting `PlaceholderText` property on a `TextBox`, the `HelpText` property stays empty, and placeholder isn't announced by screen readers. 

![A screenshot of a form with 3 text boxes, where the second TextBox has a placeholder text. Accessibility Insights app shows that HelpText property is empty on a second text box](https://user-images.githubusercontent.com/102954094/191537863-d5c3ef16-8a38-48b6-a132-9b205f88c806.png)

Narrator Buddy output when switching between these TextBoxes:

> edit, Text box 1, 
> edit, 
> edit, Text box 3, 

Placeholder text is not announced.

### Expected behavior:
When `PlaceholderText` is non-empty, `HelpText` accessible property should have value of `PlaceholderText`. As a result, it should also be announced by screen readers. 

### Steps to reproduce

Sample application:
[TextBoxPlaceholderSample.zip](https://github.com/dotnet/winforms/files/9617884/TextBoxPlaceholderSample.zip)

**Comment by Nora-Zhou01** [comment]
Verified with .NET SDK .NET 8.0.100-alpha.1.22552.10 build from main branch, this issue is fixed that When a TextBox contains placeholder text, can set the HelpText accessible property.
![AI](https://user-images.githubusercontent.com/104609169/199665761-7a0cda6d-9fc7-403e-b3b0-7e441b9bbb3d.png)
